challenges/objects-arrays.js

```diff
@@ -45,21 +69,32 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
+let universities = [];
+for(let i=0; i<graduates.length; i++){
```

Comment:
Convention is to space out operators, i.e. `for (let i = 0; i < graduates.length; i++)`. I can't highly recommend enough the VS Code extension 'Prettier', you can set it up to auto format on save, and it saves a lot of time!